workspace.spec

```diff
@@ -980,7 +980,7 @@ module Workspace {
 		UnspecifiedObject data - The object's data.
 		object_metadata metadata - Metadata for object retrieved/
 
-		@deprecated Workspaces.ObjectData
+		@deprecated Workspace.ObjectData
```

Comment:
oops

Reply:
ah, it's only the spec... not a proper code file!